cmd/broadcaster-cli/app/utxos/broadcast/broadcast.go

```diff
@@ -109,43 +103,34 @@ var Cmd = &cobra.Command{
 			return fmt.Errorf("failed to create client: %v", err)
 		}
 
-		fundingKeySets := make([]*keyset.KeySet, len(keyFiles))
-		for i, kf := range keyFiles {
+		wg := &sync.WaitGroup{}
 
-			fundingKeySet, _, err := helper.GetKeySetsKeyFile(kf)
-			if err != nil {
-				return fmt.Errorf("failed to get key sets: %v", err)
-			}
+		_, cancel := context.WithCancel(context.Background())
```

Comment:
The `ctx` variable is not created here and thus used nowhere. The `cancel()` function has no effect

Reply:
this now dealt differently